src/20_Components/amp-video.html

```diff
@@ -12,8 +15,13 @@
   <!-- ## Setup -->
   <!--
     Import the `amp-video` component.
+
+    Note that we're also including `amp-animation` for the rotate-to-fullscreen
```

Comment:
you could move the note about amp-animation above the import of amp-animation.
```
<!-- Import the `amp-video` component. -->
<script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
<!-- Import  the `amp-animation` component for the rotate-to-fullscreen sample -->
<script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js"></script>
```
then you are also importing amp-animation and amp-position observer, similar to above, you could explain why you are doing that

Reply:
These shouldn't have been added here, fixed!